src/backend/dx12/src/device.rs

```diff
@@ -1512,14 +1605,19 @@ impl d::Device<B> for Device {
         if ds_destroy { destroy_shader(ds); }
 
         if winerror::SUCCEEDED(hr) {
+            let mut baked_states = desc.baked_states.clone();
+            if !desc.depth_stencil.unwrap_or(unsafe { mem::zeroed() }).depth_bounds {
```

Comment:
similarly, let's avoid `unsafe` here, e.g. by doing `desc.depth_stencil.map(|ds| ds.depth_bounds).unwrap_or(false)`

Reply:
Thanks! Still newish to Rust and didn't know about `Option::map`.